core/domain/exp_services.py

```diff
@@ -813,15 +1034,28 @@ def update_exploration(
 
 
 def create_exploration_summary(exploration_id, contributor_id_to_add):
-    """Create summary of an exploration and store in datastore."""
+    """Create summary of an exploration and store in datastore.
+
+    Args:
+        exploration_id: str. The id of the exploration.
+        contributor_id_to_add: str or None. ID of the user to be added to
+            the contributor to the exploration.
+    """
     exploration = get_exploration_by_id(exploration_id)
     exp_summary = compute_summary_of_exploration(
         exploration, contributor_id_to_add)
     save_exploration_summary(exp_summary)
 
 
 def update_exploration_summary(exploration_id, contributor_id_to_add):
-    """Update the summary of an exploration."""
+    """Update the summary of an exploration.
+
+    Args:
+        exploration_id: str. The id of the exploration whose summary is 
+            to be updated.
+        contributor_id_to_add: str or None. ID of the user to be added as a 
```

Comment:
What if this value is None? (By the way, the grammar here is better than the docstring for the previous comment.)